temoa-uiuc/data_files/UIUC_2050_cost.sql

```diff
@@ -0,0 +1,3076 @@
+/*
+This scenario adds a transportation sector to UIUC.
+All transportation demand is measured by the energy equivalent of gasoline.
+*/
+BEGIN TRANSACTION;
+CREATE TABLE "time_season" (
+	"t_season"	text,
+	PRIMARY KEY("t_season")
+);
+INSERT INTO "time_season" VALUES ('inter');
+INSERT INTO "time_season" VALUES ('summer');
+INSERT INTO "time_season" VALUES ('winter');
+
+
+CREATE TABLE "time_periods" (
+	"t_periods"	integer,
+	"flag"	text,
+	FOREIGN KEY("flag") REFERENCES "time_period_labels"("t_period_labels"),
+	PRIMARY KEY("t_periods")
+);
+-- UIUC Data
+INSERT INTO "time_periods" VALUES(2000, 'e');
+INSERT INTO "time_periods" VALUES(2016, 'e');
+INSERT INTO "time_periods" VALUES(2020, 'e');
+INSERT INTO "time_periods" VALUES(2021, 'f');
+INSERT INTO "time_periods" VALUES(2022, 'f');
+INSERT INTO "time_periods" VALUES(2023, 'f');
+INSERT INTO "time_periods" VALUES(2024, 'f');
+INSERT INTO "time_periods" VALUES(2025, 'f');
+INSERT INTO "time_periods" VALUES(2026, 'f');
+INSERT INTO "time_periods" VALUES(2027, 'f');
+INSERT INTO "time_periods" VALUES(2028, 'f');
+INSERT INTO "time_periods" VALUES(2029, 'f');
+INSERT INTO "time_periods" VALUES(2030, 'f');
+INSERT INTO "time_periods" VALUES(2031, 'f');
+INSERT INTO "time_periods" VALUES(2032, 'f');
+INSERT INTO "time_periods" VALUES(2033, 'f');
+INSERT INTO "time_periods" VALUES(2034, 'f');
+INSERT INTO "time_periods" VALUES(2035, 'f');
+INSERT INTO "time_periods" VALUES(2036, 'f');
+INSERT INTO "time_periods" VALUES(2037, 'f');
+INSERT INTO "time_periods" VALUES(2038, 'f');
+INSERT INTO "time_periods" VALUES(2039, 'f');
+INSERT INTO "time_periods" VALUES(2040, 'f');
+INSERT INTO "time_periods" VALUES(2041, 'f');
+INSERT INTO "time_periods" VALUES(2042, 'f');
+INSERT INTO "time_periods" VALUES(2043, 'f');
+INSERT INTO "time_periods" VALUES(2044, 'f');
+INSERT INTO "time_periods" VALUES(2045, 'f');
+INSERT INTO "time_periods" VALUES(2046, 'f');
+INSERT INTO "time_periods" VALUES(2047, 'f');
+INSERT INTO "time_periods" VALUES(2048, 'f');
+INSERT INTO "time_periods" VALUES(2049, 'f');
+INSERT INTO "time_periods" VALUES(2050, 'f');
+
```

Comment:
Another possible error is that you don't have a ``2051`` year defined. The last year tells Temoa where to stop. Right now Temoa will run a simulation for 2021-2049, not 2050